LightBulb/Views/Components/Settings/GeneralSettingsTabView.axaml

```diff
@@ -156,7 +166,8 @@
                 MinWidth="24"
                 HorizontalAlignment="Right"
                 DockPanel.Dock="Right"
-                Theme="{DynamicResource CompactTextBox}">
+                Theme="{DynamicResource CompactTextBox}"
+                Classes.accent="{DynamicResource UseAccentControls}">
```

Comment:
Hmm what does this do on text box?

Reply:
Two things: it changes the selection cursor, and it changes the underline, both to the secondary color.